db/core/htrie.c

```diff
@@ -122,37 +116,25 @@ tdb_init_mapping(void *p, size_t db_size, unsigned int rec_len)
 	return hdr;
 }
 
-static inline void
+static void
 tdb_free_index_blk(TdbHtrieNode *node)
 {
 	/* Just zero the block and leave it for garbage collector. */
 	bzero_fast(node, sizeof(*node));
 }
 
 /* TODO synchronize the bucket access. */
-static inline void
```

Comment:
I am not sure that it is good idea to change all inline functions to not inline. Up to you